src/Widgets/TableWidget/tableWidget.scss

```diff
@@ -0,0 +1,185 @@
+@import "src/assets/stylesheets/variables";
+
+.table-widget {
+  margin: 20px 0;
+  max-width: 100%;
+  width: 100%;
+
+  th,
+  td {
+    vertical-align: middle !important;
+  }
+
+  .fa {
+    color: $secondary;
+    font-size: 25px;
+    line-height: 20px;
+  }
+
+  thead {
+    th {
+      font-size: 13px;
+      font-weight: 400;
+      padding: 10px 20px;
+      text-align: center;
+
+      &:first-child {
+        text-align: left;
+      }
+    }
+  }
+
+  tbody {
+    th {
+      font-size: 20px;
+      font-weight: 400;
+      padding: 20px;
+      border-top: 2px solid rgba(99, 99, 99, 0.1);
+      border-right: 2px solid rgba(99, 99, 99, 0.1);
+      border-left: 2px solid rgba(99, 99, 99, 0.1);
+    }
+
+    td {
+      font-size: 15px;
+      padding: 20px;
+      text-align: center;
+      border-top: 2px solid rgba(99, 99, 99, 0.1);
+      border-right: 2px solid rgba(99, 99, 99, 0.1);
+      width: 150px;
+
+      p {
+        margin: 0;
+      }
+
+      b {
+        font-weight: 700;
+      }
+    }
+
+    tr {
+      &:nth-child(even) {
+        th,
+        td {
+          background: rgba(200, 200, 200, 0.1);
+        }
+      }
+
+      &:hover {
+        th,
+        td {
+          background: rgba(140, 140, 140, 0.1);
+        }
+      }
+    }
+  }
+
+  tfoot {
+    th {
+      font-size: 20px;
+      font-weight: 400;
+      padding: 20px;
+      border: 2px solid rgba(99, 99, 99, 0.1);
+    }
+
+    td {
+      font-size: 14px;
+      font-weight: 700;
+      padding: 20px;
+      text-align: center;
+      border-top: 2px solid rgba(99, 99, 99, 0.1);
+      border-right: 2px solid rgba(99, 99, 99, 0.1);
+      border-bottom: 2px solid rgba(99, 99, 99, 0.1);
+    }
+
+    th,
+    td {
+      background: $secondary;
+      color: #fff;
+    }
+  }
+
+  .quantity {
```

Comment:
This is actually styling for the `PriceWidget`. Could you extract this `qunatitiy` part to a `PriceWidget.scss`, which is used by the `PriceWidget` in this PR as well? Currently `PriceWidget` can only be used in a `TableWidget`, but I don't want the code to be too much coupled.